tests/api/v1/common/testAssociations.js

```diff
@@ -90,7 +90,7 @@ function testAssociations(path, associations, joiSchema, conf) {
       .expect((res) => {
         expect(res.body).to.be.an('array');
         res.body.forEach((record) => {
-          expect(record).to.have.keys('id', 'name', assoc, 'apiLinks');
+          expect(record).to.contains.keys('id', 'name', assoc, 'apiLinks');
```

Comment:
This will be changed back after the TODO is done, right?

Reply:
Yeap